packages/bpk-docs/src/pages/PhoneInputPage/PhoneInputPage.js

```diff
@@ -29,6 +29,8 @@ import IntroBlurb from '../../components/IntroBlurb';
 import NativePhoneInput from '../NativePhoneInputPage';
 
 const DIALING_CODE_TO_ID_MAP = {
+  '1': 'us',
```

Comment:
These changes will need to be moved to the [new docs repo](http://github.com/skyscanner/backpack-docs)